src/artemis/experiment_plans/rotation_scan_plan.py

```diff
@@ -70,27 +75,63 @@ def cleanup_sample_environment(
     yield from bps.abs_set(detector_motion.shutter, 0, group=group)
 
 
+def move_x_y_z(
+    smargon: Smargon,
+    x: float | None,
+    y: float | None,
+    z: float | None,
+    wait=False,
+    group="move_x_y_z",
+):
+    if x:
+        LOGGER.info(f"x: {x}, y: {y}, z: {z}")
+        yield from bps.abs_set(smargon.x, x, group=group)
+    if y:
+        yield from bps.abs_set(smargon.y, y, group=group)
+    if z:
+        yield from bps.abs_set(smargon.z, z, group=group)
+    if wait:
+        yield from bps.wait(group)
+
+
 def move_to_start_w_buffer(
-    axis: EpicsMotor, start_angle, wait_for_velocity_set=True, wait_for_move=False
+    axis: EpicsMotor,
+    start_angle: float,
+    offset: float,
+    wait_for_velocity_set: bool = True,
+    wait_for_move: bool = False,
+    direction: RotationDirection = DEFAULT_DIRECTION,
+    max_velocity: float = DEFAULT_MAX_VELOCITY,
 ):
     """Move an EpicsMotor 'axis' to angle 'start_angle', modified by an offset and
     against the direction of rotation. Status for the move has group 'move_to_start'."""
     # can move to start as fast as possible
-    yield from bps.abs_set(axis.velocity, 120, wait=wait_for_velocity_set)
-    start_position = start_angle - (OFFSET * DIRECTION)
+    # TODO get VMAX
+    yield from bps.abs_set(axis.velocity, max_velocity, wait=wait_for_velocity_set)
+    start_position = start_angle - (offset * direction)
     LOGGER.info(
         "moving to_start_w_buffer doing: start_angle-(offset*direction)"
-        f" = {start_angle} - ({OFFSET} * {DIRECTION} = {start_position}"
+        f" = {start_angle} - ({offset} * {direction}) = {start_position}"
     )
     yield from bps.abs_set(
         axis, start_position, group="move_to_start", wait=wait_for_move
     )
 
 
-def move_to_end_w_buffer(axis: EpicsMotor, scan_width: float, wait: float = True):
-    distance_to_move = (scan_width + 0.1 + OFFSET) * DIRECTION
+def move_to_end_w_buffer(
```

Comment:
Should: We have a system test `test_move_to_end` that I think is now broken. can we fix it and maybe also not make it a system test as it could easily be a unit test

Reply:
there's already also a unit test

Reply:
My bad, missed it, thanks